### PR TITLE
Make DELTA_CALIBRATION_MENU more universal

### PR DESCRIPTION
**Pull request by thinkyhead**
The delta calibration menu is set up specifically for a radius of 90mm. This PR adds code allowing the delta calibration menu work with any radius, however it is configured.

### CODE REVIEW

File: Marlin/ultralcd.cpp [hunk 1]
```diff
@@ -1260,14 +1260,25 @@ void kill_screen(const char* lcd_msg) {
 
   #if ENABLED(DELTA_CALIBRATION_MENU)
 
+    static void _goto_tower_pos(const float &a) {
+      char cmd[20];
+      sprintf_P(cmd, PSTR("G1 X%i Y%i"), int(-(DELTA_PRINTABLE_RADIUS) * sin(a)), int((DELTA_PRINTABLE_RADIUS) * cos(a)));
+      enqueue_and_echo_commands_P(PSTR("G1 F8000 Z4\nG1 F4000"));
+      enqueue_and_echo_command(cmd);
+    }
```
Comment (Blue-Marlin) [inline Marlin/ultralcd.cpp:1268]: Why "4"? That's a case for `Z_HOMING_HEIGHT`.
Why enqueue the command? `do_blocking_move_to()` seems to be perfect here.

**Comment by ghost** [comment]
I've confirmed that "Card removed/inserted" issue has been solved.

File: Marlin/ultralcd.cpp (hunk 1 above)
Comment (thinkyhead) [inline Marlin/ultralcd.cpp:1268]: `Z_HOMING_HEIGHT` on a Delta is meaningless. The default is off, or zero, so it's not the best choice. This is for calibration, where you want the nozzle very close to the bed. I would opt for 2mm, but 4mm is almost certainly "safe" even when the delta geometry is pretty far off. I'd rather we "dictate" that this is 4mm and just say so in the documentation.

Comment (thinkyhead) [inline Marlin/ultralcd.cpp:1268]: > do_blocking_move_to

Not a bad idea…

**Pull request merged**